cpp/tests/io/json/json_whitespace_normalization_test.cu

```diff
@@ -161,4 +161,45 @@ TEST_F(JsonWSNormalizationTest, ReadJsonOption)
   CUDF_TEST_EXPECT_TABLES_EQUAL(expected_table.tbl->view(), processed_table.tbl->view());
 }
 
+TEST_F(JsonWSNormalizationTest, ReadJsonOption_InvalidRows)
+{
+  // When mixed type fields are read as strings, the table read will differ depending the
+  // value of normalize_whitespace
+
+  // Test input
+  std::string const host_input = R"(
+  { "Root": { "Key": [ { "EE": tr ue } ] } }
```

Comment:
Should we include cases like `12 34` or `12. 34` ?
I'm surprised that we only test a single case, but can't think of cases that would actually improve coverage.

Reply:
Included these cases in the same test :)